src/openvic-simulation/pop/Pop.hpp

```diff
@@ -99,7 +101,13 @@ namespace OpenVic {
 		fixed_point_map_t<Issue const*> PROPERTY(issue_distribution);
 		IndexedMap<CountryParty, fixed_point_t> PROPERTY(vote_distribution);
 
-		fixed_point_t PROPERTY(unemployment);
+		pop_size_t employed = 0;
+		constexpr fixed_point_t get_unemployment_fraction() {
+			if (!type->can_be_unemployment()) {
+				return fixed_point_t::_0();
+			}
+			return fixed_point_t { size - employed } / size;
```

Comment:
```suggestion
			return fixed_point_t::parse(size - employed) / size;
```
Safer to use `fixed_point_t::parse` to avoid any risk of treating the input as a raw value (so not shifting it up)